lib/elixir/test/elixir/protocol/consolidation_test.exs

```diff
@@ -7,7 +7,7 @@ files = Path.wildcard(PathHelpers.fixture_path("consolidation/*"))
 Kernel.ParallelCompiler.compile_to_path(files, path)
 
 defmodule Protocol.ConsolidationTest do
-  use ExUnit.Case, async: true
```

Comment:
The test "consolidation prevents new implementations" uses `ExUnit.CaptureIO.capture_io(:stderr`. 

I thought about moving that test to `warning_test.exs` too, but it required some of the setup done at the beginning of this module, and I thought moving all that setup was not a good idea.